lib/api/createJob.js

```diff
@@ -5,28 +5,44 @@ const authHelper = require("../util/auth");
 const Jenkins = require('jenkins');
 
 /**
- *  Create jenkins job
+ * Creates a job in Jenkins.
+ *
+ * @module api/createJob
+ *
+ * @param {JenkinsAuth} auth - The authentication object to pass
+ * @param {Object} jobArgs - Object that holds the job arguments.
+ * @param {string} jobArgs.repository - Git repository url
+ * @param {string} jobArgs.branch - Branch to use
+ * @param {module:api/createJob.createJobCallback} callback - Callback to call
+ *
  */
 module.exports = (auth, jobArgs, callback) => {
-  const options = authHelper.createJenkinsOptions(auth.url, auth.user, auth.password);
-  const jenkins = Jenkins(options);
-  const fileLocation = path.join(__dirname, '..', '..', 'templates', 'standard.xml');
-  const fileContent = fs.readFileSync(fileLocation, 'utf8');
-  if (fileContent) {
-    const jobTemplate = _.template(fileContent);
-    const jobXML = jobTemplate(jobArgs);
-    const jobOptions = {
-      name: jobArgs.name,
-      xml: jobXML
-    };
-    jenkins.job.create(jobOptions, function(err, data) {
-      callback(err, data);
-    });
-  } else {
-    callback("Standard template is missing!");
-  }
+    const options = authHelper.createJenkinsOptions(auth.url, auth.user, auth.password);
+    const jenkins = Jenkins(options);
+    const fileLocation = path.join(__dirname, '..', '..', 'templates', 'standard.xml');
+    const fileContent = fs.readFileSync(fileLocation, 'utf8');
+    if (fileContent) {
+        const jobTemplate = _.template(fileContent);
+        const jobXML = jobTemplate(jobArgs);
+        const jobOptions = {
+            name: jobArgs.name,
+            xml: jobXML
+        };
+        jenkins.job.create(jobOptions, function (err, data) {
+            callback(err, data);
+        });
+    } else {
+        callback("Standard template is missing!");
+    }
+
 
 };
 
+/**
```

Comment:
Is this section referencing a method?

Reply:
That's the type definition of the callback for the method above (`module.exports = function(bla)`)

In JSDoc, you use `typedef`s in the middle of no where like that as `typedef` statements are to define a type that cannot be inferred.
see http://usejsdoc.org/tags-typedef.html

Reply:
@aliok thanks for the explanation